src/XMLProvider.cs

```diff
@@ -352,17 +352,20 @@ public override string ValidateSourceSettings()
     }
 
     private string GetSourceFilePath()
-    {
-        string srcFilePath = string.Empty;
-
-        if (_sourceFileName.StartsWith(".."))
-        {
-            srcFilePath = WorkingDirectory.CombinePaths(_sourceFileName.TrimStart(new char[] { '.' })).Replace("\\", "/");
-        }
-        else
-        {
-            srcFilePath = WorkingDirectory.CombinePaths(FilesFolderName, _sourceFileName).Replace("\\", "/");
-        }
+    {        
+        // Job created in DW 10:
+        // SourceFile = /Files/source.xml
+        // SourceFile = /Files/Integration/source.xml
+        // SourceFile = /EmailIcons/source.xml
+        var srcFilePath = WorkingDirectory.CombinePaths(SourceFile).Replace("\\", "/");
+        // Job created in DW9:
```

Comment:
This DW9 stuff is not necessary, we've gone so far that we don't need to think about DW9 jobs in DW10 like this, they have to simply manually update the job for it to work.

Reply:
ok, removed